student/migrations/0002_institute_names.py

```diff
@@ -0,0 +1,35 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('student', '0001_initial'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from student.models import EducationalInstitution
+
+        institution_names = [
```

Comment:
Having one name per-line here would probably make the code more readable.

Reply:
Alright